configs/recognition/csn/README.md

```diff
@@ -0,0 +1,53 @@
+# CSN
+
+## Model Zoo
+
+### Kinetics-400
+
+|config | resolution | gpus | backbone |pretrain| top1 acc| top5 acc | inference_time(video/s) | gpu_mem(M)| ckpt | log| json|
+|:--|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
+|[ircsn_ig65m_pretrained_r152_32x2x1_58e_kinetics400_rgb.py](/configs/recognition/csn/ircsn_ig65m_pretrained_r152_32x2x1_58e_kinetics400_rgb.py)|short-side 320|8x4| ResNet152 | IG65M|80.14|94.93|x|8517|[ckpt](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/csn/ircsn_ig65m_pretrained_r152_32x2x1_58e_kinetics400_rgb/ircsn_ig65m_pretrained_r152_32x2x1_58e_kinetics400_rgb_20200803-fc66ce8d.pth)|[log](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/csn/ircsn_ig65m_pretrained_r152_32x2x1_58e_kinetics400_rgb/20200728_031952.log)|[json](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/csn/ircsn_ig65m_pretrained_r152_32x2x1_58e_kinetics400_rgb/20200728_031952.log.json)|
```

Comment:
btw what does `ir` in `ircsn` mean? inception-resnet?

Reply:
`ircsn`: Interaction-reduced channel-separated network

Reply:
Name from the paper

Reply:
should we unify the name? csn or ircsn

Reply:
I think `ircsn` is better, since there are also `ipcsn` (Interaction-preserved channel-separated bottleneck block). `ipcsn` will be added when resource permitted. (`ircsn` is higher than `ipcsn` in performance)